common/components/timeline/template.njk

```diff
@@ -0,0 +1,37 @@
+<div class="app-timeline {{- ' ' + params.classes if params.classes}}" {%- for attribute, value in params.attributes -%} {{ attribute }}="{{ value }}"{% endfor %}>
+
+  {%- for item in params.items %}
+  <div class="app-timeline__item {{- ' ' + item.classes if item.classes}}" {%- for attribute, value in item.attributes -%} {{ attribute }}="{{ value }}"{% endfor %}>
+
+    <div class="app-timeline__header">
+      <h{{ params.headingLevel | default(2) }} class="app-timeline__title {{- ' ' + item.label.classes if item.label.classes}}">
+        {{- item.label.html | safe if item.label.html else item.label.text -}}
+      </h{{ params.headingLevel | default(2) }}>
+    </div>
+
+    <div class="app-timeline__content">
```

Comment:
This can be removed as it's not being reference within the css anymore.

Reply:
Zapped